src/core/lib/security/credentials/credentials.cc

```diff
@@ -45,6 +45,14 @@ void grpc_channel_credentials_release(grpc_channel_credentials* creds) {
   if (creds) creds->Unref();
 }
 
+grpc_channel_credentials* grpc_channel_credentials_copy(
+    grpc_channel_credentials* creds) {
+  GRPC_API_TRACE("grpc_channel_credentials_copy(creds=%p)", 1, (creds));
+  grpc_core::ExecCtx exec_ctx;
```

Comment:
Why do we need a new exec_ctx?

Reply:
This is to be consistent with other similar functions, like grpc_channel_credentials_release() and the ctx will have the call-stack, good for debugging purposes.